src/Lean/Server/Rpc/RequestHandling.lean

```diff
@@ -104,7 +104,7 @@ def registerBuiltinRpcProcedure (method : Name) paramType respType
 open Lean Elab Command Term Meta in
 def registerRpcProcedure (method : Name) : CoreM Unit := do
   let env ← getEnv
-  let errMsg := "Failed to register RPC call handler for '{method}'"
+  let errMsg := s!"Failed to register RPC call handler for '{method}'"
```

Comment:
I am guessing it could even be an `m!`, if you adjust the `throwError` also.

Reply:
I could make it `m!`, but I did want to keep this PR as a very tiny bugfix! :) Would `m!` just enable hover/clickability of `method`? Or would it potentially pretty-print the name differently?

Reply:
Ok, fixed in a41bf0. :)